documentation/docs/getting-started/index.md

```diff
@@ -5,149 +5,105 @@ path: /getting-started/
-
-However, apps can gain more functionality via plugins. In the next section, we'll look at how to use a plugin.
+![Featured Image](getting_started_1.png)
 
 ---
 
-### Styling
+### Step 3: Say 'Hello World!'
+
+In the final step of this tutorial, you will rewrite the default Fusion.js demo page to make it say 'Hello World!'. To simplify this step, you must replace the demo code in the existing `home.js` file with new code that can render the 'Hello World!' text in your browser. This means the routes will remain the same and no other code changes are required.
 
-The basic Fusion.js boilerplate comes with [Styletron](https://github.com/rtsao/styletron) support installed. This is done via a plugin registration, which looks like this in `src/main.js`:
+1. Open `home.js` and remove the existing code so the file is completely empty.
+2. Copy and paste the code snippet below to `home.js`. This snippet imports the React libraries used to render the 'Hello World!' text in your browser.
 
 ```js
-// src/main.js
-import App from 'fusion-react';
-import Styletron from 'fusion-plugin-styletron-react';
-import React from 'react';
+// src/pages/home.js
 
-export default () => {
-  const app = new App(<div>Hello world</div>);
+import React from ‘react’;
 
-  app.register(Styletron);
+const Home = () => (
 
-  return app;
-};
+);
+
+export default Home;
 ```
 
-Now, let's move our `<div>` element to a separate file called `src/components/root.js` and replace the div with a styled one:
+3. Add 'Hello World!' text between `<div>` tags.
 
 ```js
-// src/components/root.js
-import React from 'react';
-import {styled} from 'fusion-plugin-styletron-react';
-
-const Panel = styled('div', {background: 'silver'});
-
-export default <Panel>Hello</Panel>;
-
-// src/main.js
-import App from 'fusion-react';
-import Styletron from 'fusion-plugin-styletron-react';
+// src/pages/home.js
 
-import root from './components/root';
+import React from ‘react’;
 
-export default () => {
-  const app = new App(root);
-
-  app.register(Styletron);
+const Home = () => (
+ <div>Hello World!</div>
+);
 
-  return app;
-}
+export default Home;
 ```
 
-The application in your browser should automatically reload to display the silver background.
-
----
-
-### Assets
+4. Hot module reloading ensures your changes will be reflected in real time if your server is still running. In your browser, you should now see your app's home page displaying 'Hello World!'
 
-Use the virtual module `assetUrl` for other asset types, such as images.
-
-```js
-// src/components/root.js
-import React from 'react';
-import {styled} from 'fusion-plugin-styletron-react';
-import {assetUrl} from 'fusion-core';
+If you don't see them, you have probably stopped the server. Re-run `yarn dev` to rerun your application. As soon the server starts again, access your file by visiting http://localhost:3000. The message `Build completed in x.xxxs` will be shown in the terminal when the server has restarted.
```

Comment:
Nitpicky: indenting here feels a little odd.  Not sure if we can fix it though... so ignore if it isn't super simple.

Reply:
Yeah, you're right. It did look odd. You can put HTML directly in the markup so added my own <ol> block to match the indentation. I think it looks a lot better.